common/src/webida/util/theme.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2012-2015 S-Core Co., Ltd.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ *
+ * @since: 15. 11. 24
+ * @author: Koong Kyungmi (kyungmi.k@samsung.com)
+ */
+
+define([
+    'external/lodash/lodash.min',
+    'webida-lib/app-config'
```

Comment:
Do you really think this way is right?
"some modules related with IDE in common dir" should be refactored
when I split out webida/util as a bower package.